CMSgov/cmcs-eregulations
# EREGCSC-1482 -- Synthetic part

Resolves [EREGCSC-1482](https://jiraent.cms.gov/browse/EREGCSC-1482)

**Description**
Creates JS file of responses to mock a synthetic part of the regs that includes all edge cases.  Adds these JSON responses as mocks to be used with storybook stories.

**This pull request changes:**

- Adds mock service worker (MSW) to storybook 
   - this fixes issues with Supplemental Content stories not working
- creates JS file of synthetic part that matches a response from V3 of the API
- Adds Synthetic Part (as Part 499) to Storybook under the "PROTOTYPE > Regulation Text" heading

> **Note**
> The MSW works locally but does not yet work on a hosted instance of Storybook. That is TODO with another story

**Steps to manually verify this change:**

1. Go to [hosted version of Storybook](https://d3qvw8g3pq7moe.cloudfront.net/storybook/index.html) for this PR
2. In the left sidebar, go to PROTOTYPE > Regulation Text
3. There should be two entries:
   - Nodes
   - Synthetic Part
4. Select Synthetic Part > Part 499 to see the entire Synthetic Part styled as expected (i.e. eRegs styling)

If you'd like to see the Mock Service Worker in action:
1. Pull down this branch locally and run `make storybook`.  Make sure the local Django dev server is not running.
2. Visit `localhost:6006` and open the DevTools console
3. Look through the noise to find `[MSW] Mocking enabled.` in red text
4. In the left sidebar, go to SUPPLEMENTAL RESOURCES > Supplemental Content > Basic
5. You should see all of the categories become active and filled with supplemental content.
6. Inspect the Network tab and notice that a call to `localhost:8000` was successful and returned 66 results even though `localhost:8000` is not currently active
